addons/popochiu/engine/templates/hotspot_template.gd

```diff
@@ -13,8 +13,8 @@ func _on_click() -> void:
 	# For example you can make the character walk to the Hotspot and then say
 	# something:
 #	E.queue([
-#		C.walk_to_clicked(),
-#		C.face_clicked(),
+#		C.queue_walk_to_clicked(),
```

Comment:
That's OK but I would take a note to rewrite templates using the simpler "await" strategy. What do you think?

Reply:
Yes... I was thinking about that too... I'll change those helper comments to the new `await` approach. Thanks for the suggestion!